cmd/soroban-cli/src/config/sign_with.rs

```diff
@@ -37,14 +46,17 @@ pub struct Args {
     /// Sign with a local key. Can be an identity (--sign-with-key alice), a secret key (--sign-with-key SC36…), or a seed phrase (--sign-with-key "kite urban…"). If using seed phrase, `--hd-path` defaults to the `0` path.
     #[arg(long, conflicts_with = "sign_with_lab", env = "STELLAR_SIGN_WITH_KEY")]
     pub sign_with_key: Option<String>,
-    /// Sign with labratory
+    /// Sign with laboratory
+    #[arg(long, conflicts_with = "sign_with_key", env = "STELLAR_SIGN_WITH_LAB")]
+    pub sign_with_lab: bool,
+    /// Lab URL for `sign_with_lab`
     #[arg(
         long,
         conflicts_with = "sign_with_key",
-        env = "STELLAR_SIGN_WITH_LAB",
-        hide = true
+        env = "STELLAR_SIGN_WITH_LAB_URL",
+        default_value = "https://lab.stellar.org/transaction/cli-sign"
```

Comment:
I suggest as a starting point we don't make this configurable and make it hardcoded. There's only one lab deployment, we don't know of anyone else running their own lab, and quickstart doesn't yet embed lab. It's easy to add this parameter in down the track once quickstart is hosting lab or if someone adds it.

Reply:
Good point, I'll remove it now